specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json

```diff
@@ -611,7 +611,7 @@
           }
         },
         "negationConditon": {
-          "description": "Describes if this is negate condition or not.",
+          "description": "Specifies negate condition or not.",
```

Comment:
This is still a verb. You can word it as "Whether this is negate condition or not"